pkg/controller/certificate-shim/helper_test.go

```diff
@@ -102,6 +105,14 @@ func Test_translateAnnotations(t *testing.T) {
 			},
 			expectedError: errInvalidIngressAnnotation,
 		},
+		"bad revision history limit": {
```

Comment:
Suggestion: add a test case that shows that annotation with value '0' gets rejected

Reply:
Thanks, added this test case